src/main/java/de/wi2020sebgroup1/cinema/entities/CinemaRoomSeatingPlan.java

```diff
@@ -29,4 +29,66 @@
 	@JoinColumn(name = "cinemaRoom_id", referencedColumnName = "id")
 	private CinemaRoom cinemaRoom;
 
+	public CinemaRoomSeatingPlan(@NotNull int seats, CinemaRoom cinemaRoom) {
```

Comment:
Hier nicht den cinemaRoom in den Konstruktor, sonst kann man kein CinemaRoom oder CinemaRoomSeatingPlan erstellen, denke ich

Reply:
done :)